inductiva/_cli/cmd_resources/cost.py

```diff
@@ -28,20 +28,21 @@ def register(parser):
 
     subparser = parser.add_parser(
         "cost",
-        help="Estimate cost of a machine in the cloud",
+        description="Estimate cost of a machine in the cloud.",
+        help="Estimate cost of a machine in the cloud.",
```

Comment:
help and description are the same?

see https://github.com/inductiva/inductiva/pull/1037#issuecomment-1924884876 on how it should be.
I'll open a task for this.